src/components/Lightbox.tsx

```diff
@@ -62,24 +79,34 @@ type LightboxStyle = {
 type Props = {
   index?: number
   onClose: () => void
+  onDelete?: (imageIndex: number) => void
   photos: string[]
   title?: string
   description?: string
   content?: JSX.Element
   style?: LightboxStyle
 }
 
-const Lightbox = ({ index, onClose, photos, title, description, content, style }: Props) => (
-  <ImageView
-    images={photos.map((photo) => ({ uri: photo }))}
-    imageIndex={index ?? 0}
-    visible={index !== undefined}
-    swipeToCloseEnabled={false}
-    onRequestClose={onClose}
-    HeaderComponent={getLightboxHeaderComponent(photos.length, onClose)}
-    FooterComponent={getLightboxFooterComponent(title, description, content, style)}
-  />
-)
+const Lightbox = ({ index, onClose, onDelete, photos, title, description, content, style }: Props) => {
+  const [imageIndexAfterSwipe, setImageIndexAfterSwipe] = useState<number>()
+
+  const imageIndex = index ?? 0
```

Comment:
Ik zou `swipe` niet in de naam van deze velden gebruiken. Dit is te specifiek en swipe-support is ook niet de verantwoordelijkheid van deze component maar van `ImageView`. Hier een suggestie:

```suggestion
  const [currentImageIndex, setCurrentImageIndex] = useState<number>()

  const initialImageIndex = index ?? 0
```

Reply:
Ja, die is wel elegant